.github/workflows/workflow-issue-status.yml

```diff
@@ -59,12 +60,27 @@ jobs:
         run: |
           ISSUE_NUMBER="${ISSUE_NUMBER}"
           OWNER="${OWNER}"
+          REPO_NAME="${{ github.repository }}"
+          FILE="${REPO_PROJECT}"
 
           declare -A PROJECT_NUMBER_ID_MAP ITEM_ID_MAP STATUS_FIELD_ID_MAP STATUS_OPTIONS_MAP
 
           echo "1. Project associated with this issue number #$ISSUE_NUMBER"
 
           PROJECT_ITEMS_JSON=$(gh issue view "$ISSUE_NUMBER" --json projectItems)
+          if [ "$(echo "$PROJECT_ITEMS_JSON" | jq '.projectItems | length')" -eq 0 ]; then
+            echo "The projectItems list is empty. Assigning project from '$REPO_PROJECT'"
+            PROJECT=$(grep "^${REPO_NAME}_" "$FILE" | cut -d'_' -f2)
```

Comment:
This is fine by me. Just wondering if the issue should just be assigned to all the projects to which the repo is associated.

Reply:
Yes, well repos are associated to "one" project

See the list inside the file `repo_project.txt`:

```txt
fertiscan-backend_FertiScan
fertiscan-frontend_FertiScan
fertiscan-pipeline_FertiScan
nachet-backend_Nachet
nachet-frontend_Nachet
nachet-model_Nachet
howard_DevSecOps
github-workflows_DevSecOps
devops_DevSecOps
finesse-backend_Finesse
finesse-frontend_Finesse
finesse-data_Finesse
```

maybe we could support multiple projects per repo, but I don't think it is necessary. In this case, if the user doesn't set a repo when creating an issue, the default project will be associated. If the user want to add another project he can do so manually